webrtc-stats.html

```diff
@@ -2884,6 +2894,70 @@ <h2>
           </section>
         </div>
       </section>
+      <section id="transceiver-dict*">
+        <h3>
+          <dfn>RTCRtpTransceiverStats</dfn> dictionary
+        </h3>
+        <p>
+          An RTCRtpTransceiverStats stats object represents an RTCRtpTransceiver
+          of an RTCPeerConnection.
+        </p>
+        <p>
+          It appears as soon as the monitored RTCRtpTransceiver object is
+          created, such as by invoking addTransceiver, addTrack or
+          setRemoteDescription. RTCRtpTransceiverStats objects can only be
+          deleted if the corresponding RTCRtpTransceiver is removed - this can
+          only happen if a remote description is rolled back.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCRtpTransceiverStats {
+    DOMString senderId;
+    DOMString receiverId;
+    DOMString mid;
+};</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCRtpTransceiverStats</a> Members
+            </h2>
+            <dl data-link-for="RTCRtpTransceiverStats" data-dfn-for="RTCRtpTransceiverStats" class=
+            "dictionary-members">
+              <dt>
+                <dfn><code>senderId</code></dfn> of type <span class=
+                "idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  The identifier of the stats object representing the
+                  RTCRtpSender associated with the RTCRtpTransceiver represented
+                  by this stats object.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>receiverId</code></dfn> of type <span class=
+                "idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  The identifier of the stats object representing the
+                  RTCRtpReceiver associated with the RTCRtpTransceiver
+                  represented by this stats object.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>mid</code></dfn> of type <span class=
+                "idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  If the RTCRtpTransceiver that this stats object represents has
+                  a <code>mid</code> value that is not null, this is that value,
```

Comment:
should we say that the mid is extracted from the SDP?

Reply:
I'll point to [[WEBRTC-PC]]